tests/testpeeridauth.nim

```diff
@@ -0,0 +1,57 @@
+{.used.}
+
+# Nim-Libp2p
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+{.push raises: [].}
+
+import json, uri
+import chronos
+import chronos/apps/http/httpclient
+import ../libp2p/[stream/connection, upgrademngrs/upgrade, peeridauth, wire]
+
+import ./helpers
+
+const
+  AuthPeerURL = "https://registration.libp2p.direct/v1/_acme-challenge"
```

Comment:
`AuthPeerURL` why is this not default value used in `send`? would it be required to redefined this variable in other places: 1st) in test 2nd) component that uses `PeerIDAuthApi` (could be many

Reply:
> AuthPeerURL why is this not default value used in send? 

Because one can use `peeridauth` with several different applications, this is not something exclusive to autotls. I'm just using the autotls broker to test it here.